components/Calendar/config.js

```diff
@@ -0,0 +1,28 @@
+import { GITHUB_ORG } from '../../lib/settings'
+
+export const placeholderRepos = {
+  newsletters: [
+    {
+      publicationDays: [1, 2, 3, 4, 5],
+      publicationTime: '4:30',
+      repoId: `${GITHUB_ORG}/template-www`
+    },
+    {
+      publicationDays: [1, 2, 3, 4, 5],
+      publicationTime: '19:00',
+      repoId: `${GITHUB_ORG}/template-covid-19-uhr-nl`
+    },
+    {
+      publicationDays: [6],
+      publicationTime: '4:30',
+      repoId: `${GITHUB_ORG}/template-weekend`
+    }
+  ],
+  articles: [
+    {
+      publicationDays: [3],
+      publicationTime: '4:30',
+      repoId: `${GITHUB_ORG}/template-am`
+    }
+  ]
+}
```

Comment:
@patrickvenetz would you mind considering if there is a better solution for this? `repoId` refers to template repo ids and shouldn't be the same in staging and production.